pkg/resources/samples/trace-seccomp.yaml

```diff
@@ -3,12 +3,14 @@ kind: Trace
 metadata:
   name: seccomp
   namespace: gadget
+  labels:
+    trace-instance: "test123"
+    trace-node: "minikube"
```

Comment:
Generally speaking: can't we generate the seccomp profile operator labels from the trace spec?, for instance

trace-instance: `trace.ObjectMeta.Namespace` + `trace.ObjectMeta.Name`
trace-node: `trace.Spec.Node`

Reply:
I'll add this information in the seccomp profile annotation.